### PR TITLE
Switch java symlink creation to use CMD to avoid PS upgrade

### PR DESCRIPTION
**Pull request by sxa555**
Signed-off-by: Stewart Addison <sxa@uk.ibm.com>

### CODE REVIEW

File: ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Java12/tasks/main.yml [hunk 1]
```diff
@@ -28,20 +28,10 @@
...
-  win_shell: New-Item -Path 'C:\openjdk\jdk-12' -ItemType SymbolicLink -Value ([Environment]::GetEnvironmentVariable("Java12_Folder", "Machine"))
+- name: Create symlink to directory without spaces if not already there
+  win_shell: for /D %a in ("C:\Program Files\Java\jdk-12*") do IF NOT EXIST "C:\openjdk\jdk-12" MKLINK /D "C:\openjdk\jdk-12" "%a"
   args:
-    executable: powershell
-  when: (java12_symlink.stat.exists == false)
-  tags: Java12
+    executable: cmd.exe
+    creates: 'C:\openjdk\jdk-12'
+  tags: Java11
```
Comment (karianna) [inline ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Java12/tasks/main.yml]: correct tag?

File: ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Java8/tasks/main.yml [hunk 2]
```diff
@@ -28,22 +28,11 @@
...
-
-- name: Get Java 8 Folder name to create symlink
-  win_shell: powershell ([Environment]::SetEnvironmentVariable("Java8_Folder", (Get-ChildItem -Path 'C:\Program Files\Java\jdk8*' | Select-String -Pattern 'C*'), "Machine"))
-  when: (java8_symlink.stat.exists == false)
-  tags: Java8
-
-- name: Create symlink to Java 8
-  win_shell: New-Item -Path 'C:\openjdk\jdk-8' -ItemType SymbolicLink -Value ([Environment]::GetEnvironmentVariable("Java8_Folder", "Machine"))
+- name: Create symlink to directory without spaces if not already there
+  win_shell: for /D %a in ("C:\Program Files\Java\jdk8*") do IF NOT EXIST "C:\openjdk\jdk-8" MKLINK /D "C:\openjdk\jdk-8" "%a"
```
Comment (karianna) [inline ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Java8/tasks/main.yml]: `C:\Program Files\Java\jdk8*` -> `C:\Program Files\Java\jdk-8*`  ? Not sure.

Comment (sxa555) [reply ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Java8/tasks/main.yml]: Target path is `C:\openjdk\jdk-8` for consistency with other versions in our space-free path names, the directory under `c:\Program Files\Java` comes from the directory inside our tarballs which is `jdk8u212...` for JDK8 and `jdk-9...` for 9 and above so I believe this is correct

File: ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Java12/tasks/main.yml (hunk 1 above)
Comment (sxa555) [reply ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Java12/tasks/main.yml]: Nope - fixed. Looks like I'd fixed that on my filesystem but not pushed it back.

**Pull request merged**